pubspec.yaml

```diff
@@ -15,8 +15,8 @@ dependencies:
 
   enough_html_editor:
     git:
-      url: https://github.com/linagora/enough_html_editor.git
-      ref: email_supported
+      url: https://github.com/aryan-more/enough_html_editor.git
+      ref: "patch-1"
```

Comment:
please wait for PR in `enough_html_editor` merge.

Reply:
We don't need to upgrade it

Reply:
@dab246 we need to upgrade that because enough_html_editor old package are not compatible with flutter 3.16.0

Reply:
I understand what you mean, but we only make changes for debugging purposes. When merging into the TMail project, we only need to upgrade `enough_html_editor`

Reply:
@dab246 yes i know that i will change the source of dependency as soon as enough_html_editor branch email-supported get merged to it.BTW i also changed flutter version to 3.16.0 in workflow ,so it workflow don't fail .